src/algebra/category/Module/change_of_rings.lean

```diff
@@ -110,7 +110,7 @@ Extension of scalars is a functor where an `R`-module `M` is sent to `S ⊗ M` a
 `l : M1 ⟶ M2` is sent to `s ⊗ m ↦ s ⊗ l m`
 -/
 def map' {M1 M2 : Module.{v} R} (l : M1 ⟶ M2) : (obj' f M1) ⟶ (obj' f M2) :=
-@linear_map.base_change R S M1 M2 _ _ ((algebra_map S _).comp f).to_algebra _ _ _ _ l
+by apply (@linear_map.base_change R S M1 M2 _ _ ((algebra_map S _).comp f).to_algebra _ _ _ _ l)
```

Comment:
I'm all for this if it speeds things up and doesn't make the definition harder to use, but an inline comment would probably be helpful for future readers.

bors d+

Reply:
I don't know how to explain the speedup properly, so I just added a comment stating it matter-of-factly. Improvements are appreciated!